README.md

```diff
@@ -1,50 +1,65 @@
-# Ethereum point-of-sale
+![Logo](./public/favicon.ico)
 
-A vendor-facing point-of-sale React app to view incoming transactions to an Ethereum wallet.
+# Ethereum POS
+> Accept ERC-20 token payments in-store.
+
+Allows shops to accept ERC-20 compliant cryptocurrency payments.
 
 [![CircleCI](https://circleci.com/gh/HeathWallace/ethereum-pos.svg?style=shield&circle-token=4090d7f93c1bd552f529296e4cf5503cb22e44ed)](https://circleci.com/gh/HeathWallace/ethereum-pos)
 
-## Getting Started
+## Installing / Getting started
+
+A quick introduction of the minimal setup you need to get the app up & running.
+
+```shell
+git clone git@github.com:HeathWallace/ethereum-pos.git
+cd ethereum-pos
+npm install
+npm start
+```
+
+Create a file `.env` in the root of the app and add the variables shown in the console.
```

Comment:
Could this step actually be automated...? Or is that too risque?

Reply:
The issue in automating this in the build promotes Gromits over anything else. (Where the application should work with anything) - We also want to keep our internal identity resolver separate, etc.